### PR TITLE
Article Digital grundkurs i klimatanpassning 15-16 nov 2023

### LINKED ISSUE #95: Write article on "Digital grundkurs om klimatanpassning"
We should write an article about participation in the SMHI-led course "Digital grundkurs om klimatanpassning", and include links to klimatanpassning.se and the course.

### PR DESCRIPTION
**Pull request by docNord**
Write article on the SMHI-led course from klimatanpassning.se for publication on the home page.

- [x] Descriptive text
- [x] Link to klimatanpassning.se and course syllabus

Closes #95

### CODE REVIEW

**Review by mgcth — APPROVED** [review]
I think this text summarises it well as is! Thanks for writing it :)

Just a question about category, but that can be changed later it needed.

File: content/posts/digital-grundkurs-klimatanpassning2023-11-15.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+title: Deltagande vid Digital Grundkurs i Klimatanpassning
+date: 2023-11-15
+modified:
+author: Anders Nord
+category: Projekt
```
[the remark this answers is not in the document]
Comment (docNord) [reply content/posts/digital-grundkurs-klimatanpassning2023-11-15.md:5]: What other tag is there? 🤷‍♂️

[the remark this answers is not in the document]
Comment (mgcth) [reply content/posts/digital-grundkurs-klimatanpassning2023-11-15.md:5]: We don't have any now, but could consider more tags as this isn't really a project, or maybe it is?

[the remark this answers is not in the document]
Comment (docNord) [reply content/posts/digital-grundkurs-klimatanpassning2023-11-15.md:5]: I agree. I also think that we should consider showcasing the "project" articles somehow, as they are typically more worked through than the aw notifications and represent the bulk work of the organisation. Perhaps we should start an issue or two on the subject?

[the remark this answers is not in the document]
Comment (mgcth) [reply content/posts/digital-grundkurs-klimatanpassning2023-11-15.md:5]: Sure, open issue about it!

**Pull request merged**